scenarios/appsec/test_traces.py

```diff
@@ -12,7 +12,7 @@
 
 
 @rfc("https://datadoghq.atlassian.net/wiki/spaces/APS/pages/2365948382/Sensitive+Data+Obfuscation")
-@released(golang="1.38.0", dotnet="?", java="?", nodejs="?", php_appsec="0.3.0", python="?", ruby="?")
+@released(golang="1.38.0", dotnet="?", java="0.100.0", nodejs="?", php_appsec="0.3.0", python="?", ruby="?")
```

Comment:
it means that as soon as your master branch build this version number, it means that the test must be ok, otherwise it will break your CI, ok for you ?

Reply:
This has been merged already for 0.100.0